### DISCUSSION TITLE
Are you speaking about Green Software?

### DISCUSSION THREAD

**Discussion by NAMRATA-WOKE** [Green-Software-Foundation/discussions#22] [Events]
**Have you recently spoken about green software at an industry event or conference? Do you have one coming up? Let us know!** 💚

We are committed to celebrating the efforts of our software practitioners to make technology more sustainable and software greener. We invite you to share your latest talks on Green Software and our projects with us! Doing so allows us to amplify your contribution to educate, inform and inspire the transition to greener software practices across our marketing and communication channels.

You can drop your details here or [complete this form](https://docs.google.com/forms/d/e/1FAIpQLScdlrF3-6IRQt-5mGELyxHeNgwUvnQFmRiuBze4jow7gd7y9g/viewform?usp=sf_link). 

Do you know what we know? Do we know what you know? Check out our [Events Tracker](https://docs.google.com/spreadsheets/d/1JIfSXcOXkvPXozWEAOowoBjevKpB770l_CFdLikjzNA/edit?usp=sharing)!

**Comment by chrisxie-fw** [comment]
Thank you @NAMRATA-WOKE for initiating this discussion. I have recently had a keynote speech at LF Energy Summit Paris 2023 on Energy Sustainability from ICT's Perspective, in which I mentioned our work with GSF. Here is the [linkedIn post to learn more](https://www.linkedin.com/posts/chrisxie-us_keynote-a-vision-of-energy-sustainability-activity-7084965362923929601-zeGD?utm_source=share&utm_medium=member_desktop). The YouTube video of the presentation is here: https://lnkd.in/g_kgepMw
Here is the slides for a deeper dive into the topics discussed: https://lnkd.in/gfFE6j69

**Reply by NAMRATA-WOKE** [reply]
Thanks for sharing, Chris!

**Comment by ines-akrap** [comment]
Hi, 

as I mentioned in my introduction in 'Welcome' section, I consider topic of Sustainable Web Design very interesting and important and am trying to raise awareness about it by talking at the conferences. 

I gave my first talk about the topic on [Google WTM Event in March 2021](https://www.youtube.com/watch?v=K7xTy6IUAi8&t=8s).
I also talked about the topic on recent Google WTM Event in London this year and together with my colleague Claire Thornewill at [Linux Foundation Energy Summit](https://www.youtube.com/watch?v=UTXVsVqh4kE&list=PL9JMtRpKSiZiX6PE-aFWeqijNbEFflrpq&index=3&pp=gAQBiAQB) few weeks back.

I will be presenting my talk "Reducing the carbon footprint of your website" next week at [WeAreDevelopers World Congress in Berlin](https://www.wearedevelopers.com/world-congress/program) and in September at [Infobip Shift Conference in Croatia](https://shift.infobip.com/speakers/). 

Hopefully this helps us on the road to more sustainable Web :)

**Reply by NAMRATA-WOKE** [reply]
@ines-akrap can you share more about your Google WTM Event talk with Claire Thornewill? I'm tracking the # of talks on green software and would love to include it in our accounting! We've included your talk at Infobip in our [Events Tracker](https://docs.google.com/spreadsheets/d/1JIfSXcOXkvPXozWEAOowoBjevKpB770l_CFdLikjzNA/edit?usp=sharing)!

**Reply by ines-akrap** [reply]
@NAMRATA-WOKE 

So, in March, both Claire and myself had a (separate) talks on Google London IWD. Claire did "Sustainable Web Design" and I did "Reducing the carbon footprint of your website". 
In June we had a joint talk "Designing Energy Efficient Websites" at Linux Foundation Energy Summit. 

We work together and complement each other nicely since Claire comes from business and design background and myself from engineering, so she she is usually the one talking about whys and all the important impacts, and I am the one covering technical part (how). 

She also did few more talks this year and has few upcoming ones, but I will let her provide you with her full list :)

**Reply by mkorbi** [reply]
Hi @ines-akrap maybe you would like to give a talk one day on our GSF Munich group? https://www.meetup.com/gsf-munich/

**Reply by adamj89** [reply]
@mkorbi I had a chat with @ines-akrap and @cthornewill about getting involved with Munich. Should we schedule a quick call?

**Comment by cthornewill** [comment]
Hi @NAMRATA-WOKE,
thanks for the initiative, here is a list of my talks I have given on the topic:

1. 2021 - [Start Your Journey In Creating Sustainable Web Designs](https://uxdx.com/session/rec7D29bHBDZzBKZG) - UXDX Community Event
2. IWDLondon 2023 - Sustainable Web Design - [Women Tech Makers](https://gdg.community.dev/events/details/google-gdg-london-presents-women-techmakers-international-womens-day-daretobe/)
3. 17.05.2023 - [TEQnation 2023 - Sustainable Web Design](https://www.youtube.com/watch?v=8YrqlcmyYE0&t=1091s) 
4. 1-2.06.2023 - [LF Energy Summit 2023 - Designing Energy Efficient Websites](https://www.youtube.com/watch?v=UTXVsVqh4kE) with @ines-akrap 
5. 15.06.2023 - [Green Digital Marketing](https://www.youtube.com/watch?v=a5MjHAvlgts&t=391s) for the INCO Academy Green Digital Skills Speaker Series
6. 6-7.9.2023 - Fireside Chat: Powering the Future: Envisioning Sustainable Technology and Design at [Sustainability Live London](https://sustainability-live.com/) 
7. 2-4.10.2023 - Impacting Climate Change One Byte at a Time - [GoTo Copenhagen 2023](https://gotocph.com/2023)

**Reply by NAMRATA-WOKE** [reply]
This is amazing! Thank you @cthornewill. I had a few noted down but will make sure to add the rest to our tracker!

**Comment by dgeat** [comment]
Hello everyone, My colleague and I are running the following workshop : https://iuk.ktn-uk.org/events/chatgpt-generative-ai-and-large-language-models-workshop/ on September 18th in Newcastle-Upon-Tyne (Northeast of the UK). **We have a 15 minute speaker slot** (and a table for a demonstration if wanted) and we would love for someone to come and talk about **the environmental impact of generative AI (in particular)**.  The event is aimed at loads of different types of organisations including SMEs and government departments. If you are keen to be the speaker, or know of someone who might be, please would you  message me via [LinkedIn](https://www.linkedin.com/in/dlg483/). Thank you!

**Reply by NAMRATA-WOKE** [reply]
@dgeat thanks for sharing this opportunity!

**Comment by ikolaxis** [comment]
Hi @NAMRATA-WOKE ,

I will speak at the following conferences about "Green Software: Rethinking how to develop software to save our planet":

- 19 October 2023, [BaselOne](https://baselone.ch/one), Basel, Switzerland: [Session details](https://baselone.ch/speech.html?id=17C65456-1096-4180-AB21-EC50A89FEB97)
- 10 November 2023, [Open Conf](https://www.open-conf.gr/), Athens, Greece: [Session details](https://www.open-conf.gr/etn-speaker/ioannis-kolaxis/)

Please let me know if you need any additional information.

Thanks,
Ioannis

**Comment by olegoaer** [comment]
Hi @NAMRATA-WOKE 

I am [member](https://speakers.greensoftware.foundation/speakers/olivier-le-goaer/) of the Green Software Foundation's Speaker's Bureau and I just gave a keynote at Microsoft France ([event here](https://msevents.microsoft.com/event?id=1496731230))

Here is the slides for a deeper dive into the topic discussed : [The road to green code](https://olegoaer.perso.univ-pau.fr/works/The_road_to_green_code.pdf)

Thanks

**Reply by NAMRATA-WOKE** [reply]
@olegoaer wonderful, thank you so much! If you'd like to write about the experience or need our help turning it into a blog for the GSF website, let me know. 

You can propose a new article issue on GitHub 👉🏼 https://github.com/Green-Software-Foundation/writers

Be sure to share your email with me at namrata@greensoftware.foundation so I can follow-up with you!

**Reply by olegoaer** [reply]
Yes, I'm planning to write an article for GSF. Stay tuned ;)

**Reply by NAMRATA-WOKE** [reply]
@olegoaer great! Do share it once it's ready!

**Comment by mistrc** [comment]
It is probably too long ago now, but just in case.  Last year, in December, I was part of an internal even at GlobalLogic (UK&I division) and spoke about Green Software.  I am no longer with the firm, but if it is not too old, they could be contacted for the video of the talk.

We had split the talk into three parts,
1. Talk from our sister company Method, on UX/UI principles for green software and the initiatives that they are undertaking
2. Talk from myself to talk about the coding effort that had been taken and the thought processes for making code better as well as running on different hardware suited to the task
3. A challenge for the audience to take a site that we had deliberately made less green to get as high a score as they could in https://ecograder.com/

**Comment by danielvaughan** [comment]
Hi, I am a member of the Speakers Bureau and have two Green Software sessions this week.

On Wednesday, I am speaking at IBM's Z Day conference with a talk titled "Green Software Development for a Sustainable Future". Link here but it is only available after registration: https://ibmzday2023-vconf.bemyapp.com/#/talks/65258a6a120f3ec7897bb8b6

On Thursday, I am facilitating a discussion on Green Engineering at Wave Talent's Engineering Leaders Workshop: in London: https://www.eventbrite.co.uk/e/making-waves-the-engineering-leaders-workshop-take-2-tickets-680783330267?aff=oddtdtcreator

**Comment by obierlaire** [comment]
Hi @NAMRATA-WOKE ,

Over the past months, I talked about carbon emissions of the cloud infrastructure in the following conferences: 
- Devoxx Belgium [Toward a carbon-aware Cloud](https://devoxx.be/talk/?id=36654) - [Video](https://www.youtube.com/watch?v=WvoRz5EYR7Y&t=1335s) - 4 Octobre 2023
- ChefConf Munich, Germany [agenda](https://www.chef.io/chefconf/munich/) - 12 Octobre 2023
- NDC Porto, Portugal [Toward a carbon-aware Cloud](https://ndcporto.com/agenda/toward-a-carbon-aware-cloud-0gks/05yrkhh6ail) - [Live feed (around 1:36:03)](https://www.youtube.com/live/xh78dBLsV3U?si=YkqvgUCBD46rtYud) - 18 Octobre 2023
- Devfest Nantes, France [Towards a 'carbon-aware' cloud](https://devfest.gdgnantes.com/en/sessions/towards_a__carbon_aware__cloud_/) - [Video](https://youtu.be/RCb9peXqRfM?feature=shared) - 19 October 2023
- All Day Devops (online) [Toward a Carbon-Aware Cloud](https://www.alldaydevops.com/addo-speakers/olivier-bierlaire) - [Replay On demand](https://www.alldaydevops.com/register-2023) - 26 October 2023

And I'll give this talk next: 
- 21 Novembre at [JCON World ](https://sched.co/1VD62) (Online)
- 26 November at [Devops D-Day](https://www.devopsdday.com/) in Marseille, France
- 7 Dec 2023 - [HashiTalks: Deploy](https://events.hashicorp.com/hashitalksdeploy) (online)

I had a few questions after each of those talks:
- Why should it be our responsibility as developers/sre, shouldn't it be cloud provider's responsibility to reduce their emissions? 
- How can you calculate emissions of managed services (AWS S3, lambda functions, firebase...)? 
- How many companies need to report their CO2 emissions of their cloud?

**Reply by NAMRATA-WOKE** [reply]
@obierlaire Apologies for not responding sooner - thank you for sharing your 2023 talks, and please do let us know of upcoming talks in 2024. I wanted to ask you to elaborate on the first question you asked above, and if you could share a bit of context behind it.
